test/test_app.cpp

```diff
@@ -6,9 +6,9 @@
  *  SPDX-License-Identifier: LGPL-3.0-or-later
  ****************************************************************************/
 
-
 #include "app/app.h"
 
+#include <array>
```

Comment:
is that really needed due to the version change?

Reply:
Yes; it used to be transitively included, which is no longer the case in GCC12.